ivy/functional/ivy/general.py

```diff
@@ -2629,6 +2629,26 @@ def scatter_nd(
     -------
     ret
         New array of given shape, with the values scattered at the indices.
+    
+    Examples
+    --------
+    scatter values into a zeros array
+    
+    >> indices = ivy.Array([4, 3, 1, 7])
+    >> updates = ivy.Array([9, 10, 11, 12])
+    >> shape = ivy.Array([8])
+    >> scatter = ivy.scatter_nd(indices, updates, shape)
+    >> print(scatter)
+    [0, 11, 0, 10, 9, 0, 0, 12]
```

Comment:
Outputs should be of type `ivy.array`, not list. please update accordingly. Also, please provide more docstring examples with other types of inputs, e.g. `ivy.Container` or a mix of `ivy.Array` and`ivy.Container`.